src/elements/table/table-wrapper/table-wrapper.scss

```diff
@@ -0,0 +1,33 @@
+@use '../../core/styles/index' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  display: block;
+}
+
+:host([data-has-horizontal-scrollbar]) {
+  ::slotted(.sbb-table) {
+    // Remove the border-radius from the bottom of the table if the scroll is visible
+    --sbb-table-border-radius: var(--sbb-border-radius-4x) var(--sbb-border-radius-4x) 0 0 !important;
+  }
+}
+
+::slotted(.sbb-table) {
+  width: 100%;
+}
+
+.sbb-table-wrapper {
+  overflow: auto;
+  @include sbb.scrollbar($size: thick, $track-visible: true);
+
+  &::-webkit-scrollbar {
+    border-radius: 0 0 var(--sbb-border-radius-4x) var(--sbb-border-radius-4x);
```

Comment:
As it is used 4 times, should we not create an addition var for the border-radius-4x? like --sbb-table-wrapper-border-radius?